src/component/atom/SearchIcon/searchIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { IconButton } from "@mui/material";
+import SearchOutlinedIcon from "@mui/icons-material/SearchOutlined";
+import { FC, useContext } from "react";
+import { NavbarMenuContext } from "../../../context/navbarMenuContext";
+
+const SearchIcon: FC = () => {
+  const { toggleSearchButton } = useContext(NavbarMenuContext);
+  return (
+    <IconButton
+      aria-label="cart"
+      onClick={() => {
+        toggleSearchButton(false);
```

Comment:
it is not good name of this function. Toggle is something that you do not need to set by passing true/false it should automatically do that instead. Please change that.